### PR TITLE
Detect access attempts to sensitive data

### PR DESCRIPTION
**Pull request by erenJag**
:warning: This PR depends on https://github.com/crowdsecurity/sec-lists/pull/1/files

### CODE REVIEW

File: scenarios/crowdsecurity/http-sensitive-files.yaml [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+format: 2.0
+debug: true
+name: crowdsecurity/http-sensitive-files
+description: "Detect attempt to access to sensitive files (.log, .db ..) or folders (.git)"
+filter: 'evt.Meta.service == "http" and any(File("sensitive_data.txt"), { evt.Parsed.request endsWith #})'
+groupby: "evt.Meta.source_ip"
+data:
+   - source_url: https://raw.githubusercontent.com/crowdsecurity/sec-lists/master/web/sensitive_data.txt
+   - dest_file: sensitive_data.txt
+capacity: 2
```
Comment (buixor) [inline scenarios/crowdsecurity/http-sensitive-files.yaml:11]: I think we want to distinct on the URI to limit the impact of false positives :)

File: scenarios/crowdsecurity/http-sensitive-files.yaml [hunk 2]
```diff
@@ -0,0 +1,17 @@
+type: leaky
+format: 2.0
+debug: true
+name: crowdsecurity/http-sensitive-files
+description: "Detect attempt to access to sensitive files (.log, .db ..) or folders (.git)"
+filter: 'evt.Meta.service == "http" and any(File("sensitive_data.txt"), { evt.Parsed.request endsWith #})'
```
Comment (buixor) [inline scenarios/crowdsecurity/http-sensitive-files.yaml:6]: as well I think the filter is too lose, we should check that it's access or error log no ?

Comment (erenJag) [reply scenarios/crowdsecurity/http-sensitive-files.yaml]: done :+1:

File: scenarios/crowdsecurity/http-sensitive-files.yaml (hunk 1 above)
Comment (erenJag) [reply scenarios/crowdsecurity/http-sensitive-files.yaml:12]: umm, I not sure, if we add `distinct` and an attacker try to access at least 3 different (files and directories in the list) it's just no ?

Comment (erenJag) [reply scenarios/crowdsecurity/http-sensitive-files.yaml]: After reflection, you're right.

**Pull request merged**